modules/cudf/src/data_frame.ts

```diff
@@ -112,20 +213,55 @@ export class DataFrame<T extends TypeMap = any> {
    * Return a new DataFrame with specified columns removed.
    *
    * @param names Names of the columns to drop.
+   *
+   * @example
+   * ```typescript
+   * import {DataFrame, Series, Int32, Float32}  from '@rapidsai/cudf';
+   * const df = new DataFrame({
+   *  a: Series.new({type: new Int32, data: [0, 1, 1, 2, 2, 2]}),
+   *  b: Series.new({type: new Float32, data: [0, 1, 2, 3, 4, 4]})
+   * });
+   *
+   * df.drop(['a']) // returns df {b: [0, 1, 2, 3, 4, 4]}
+   * ```
    */
   drop<R extends keyof T>(names: R[]) { return new DataFrame(this._accessor.dropColumns(names)); }
 
   /**
    * Return whether the DataFrame has a Series.
    *
    * @param name Name of the Series to return.
+   *
+   * @example
+   * ```typescript
+   * import {DataFrame, Series, Int32, Float32}  from '@rapidsai/cudf';
+   * const df = new DataFrame({
+   *  a: Series.new({type: new Int32, data: [0, 1, 1, 2, 2, 2]}),
+   *  b: Series.new({type: new Float32, data: [0, 1, 2, 3, 4, 4]})
+   * });
+   *
+   * df.has('a') // true
+   * df.has('c') // false
+   * ```
    */
   has(name: string) { return this._accessor.has(name); }
 
   /**
    * Return a series by name.
    *
    * @param name Name of the Series to return.
+   *
+   * @example
+   * ```typescript
+   * import {DataFrame, Series, Int32, Float32}  from '@rapidsai/cudf';
+   * const df = new DataFrame({
+   *  a: Series.new({type: new Int32, data: [0, 1, 1, 2, 2, 2]}),
+   *  b: Series.new({type: new Float32, data: [0, 1, 2, 3, 4, 4]})
+   * });
+   *
+   * df.get('a') // Int32Series { _col: Column {} }
+   * df.get('b') // Float32Series  { _col: Column {} }
```

Comment:
Is the `{ _col: Column {} }` useful to users? (For that matter, can we make the "repro" look better in general for series)

Reply:
removed `{ _col: Column {} }`